packages/python/plotly/test_requirements/requirements_39_pandas_2_optional.txt

```diff
@@ -19,3 +19,5 @@ matplotlib==2.2.3
 scikit-image==0.18.1
 psutil==5.7.0
 kaleido
+vaex
```

Comment:
Why do you not add modin here?

Reply:
`test_build_df_using_interchange_protocol_mock` test should be enough for modin. On the other hand, I do not want to expand the list of dependencies, there are already enough of them.